src/graphql/data/util.tsx

```diff
@@ -93,6 +93,20 @@ const URL_CHAIN_PARAM_TO_BACKEND: { [key: string]: Chain } = {
   optimism: Chain.Optimism,
 }
 
+export const CHAIN_ID_TO_URL_NAME: Record<SupportedChainId, string> = {
```

Comment:
this maybe should live in a different file also. it is starting to not deal with graphql specific things

Reply:
i will deal with moving this later once i migrate the token details page to also use this constant. lmk if you have a better idea for the naming!